Library/Formula/amdatu-bootstrap.rb

```diff
@@ -0,0 +1,11 @@
+class AmdatuBootstrap < Formula
+  desc "Bootstrapping OSGi development"
+  homepage "http://amdatu.org/bootstrap/intro.html"
+  url "https://bitbucket.org/amdatu/amdatu-bootstrap/downloads/bootstrap-r8-bin-brew.zip"
```

Comment:
This one contains the amdatu-bootstrap.sh script that I added for Brew. For next releases that will be included directly in the release.